doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+# Compressed Transaction Schema
+
+## 1. Abstraction
+With this transaction compression schema we reduce the size of all transactions by at least 15-20% and up to 40% with signature recovery. The main current uses for this will be steganography or other low bandwidth channels for passing transaction data.
+
+## 2. Methods
+The four main methods to achieve a lower transactions size are:
+1. Adding a few bytes of data before the transaction and each of its inputs and outputs to determine the structure of the following data.
+2. Removing all excess white space and fillers with the use of Variable Integers.
+3. The use of compressed signatures and public key recovery upon decompression
+4. Replacing the 32 byte Txid with a block height and index for the transaction instead.
+
+## 2. Schema
+
+| Name                           | Value     | Description |
+| -------------------------------|-----------|-------------|
+| Control VarInt(CV)             | 1-2 Bytes | A Variable Integer containing information on the structure of the transaction. |
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

Need to specify what every bit is used for; can do this after the table. Similar for other control byte fields.

Reply:
Shouldn't call this a varint because it's not an integer. Can call it a "control byte" even though it might be 2 bytes.

Reply:
Fixed Thanks!